docs/Makefile

```diff
@@ -39,6 +39,7 @@ apidocs:
 	cd ../sdk && npm ci && npm run typedoc
 	mkdir -p "${managed_attachments}"
 	rsync -a ../sdk/apidocs/ "${managed_attachments}/api/"
+	bin/version.sh | xargs -0  printf "%s" > "${managed_attachments}/latest-version.txt"
```

Comment:
Why is the printf needed, to strip a linefeed?

Reply:
Good point. It's not needed.